CuratedContent/CMakeTutorialAndReferenceResources.md

```diff
@@ -0,0 +1,87 @@
+Web links | https://crascit.com/professional-cmake/
+
+The CMake meta-build tool and ecosystem (which includes the tools CTest and CPack with each CMake installation) continues to evolve and improve fairly rapidly from release to release.
+Kitware (the company that owns the primary CMake repository and provides the official releases of CMake) puts out three major releases of CMake each year (or about every four months).
+The best practices for using CMake evolve as new versions of CMake are released.
+(For example, at the time of this writing, CMake support for a feature called Generator Expressions is still rapidly evolving and can have a dramatic impact on the best way to write CMakeList.txt files.)
+Therefore, unlike other programming tools, trying to use web searches to find solutions for CMake can be ineffective; because even advice from just a year ago on Stack Overflow or Stack Exchange can be outdated.
+Therefore, this page gives advice based on 13+ years of CMake usage and hundreds of hours of searching for and reading CMake documentation, especially in the years 2021-2022.
+
+<img src='../images/ProfessionalCMake.jpg' class='page'/>
+
+Arguably, the best resource for finding up-to-date guidance on how to use CMake is the most recent edition of the book ***Professional CMake*** (the 11th edition covering CMake versions up to 3.22 as of this writing).
+The 560+ page 11th edition of the book contains excellent tutorial material for CMake and yet goes into deep concepts in the CMake language where needed (such as the gotchas and workarounds for dealing with lists and function/macro arguments with data containing quotes `"`, spaces ` ` and semi-colons `;` which is one of the most confusing aspects to using CMake).
+This book covers every basic CMake feature in great detail.
+It explains modern CMake idioms and best practices.
+Also, it covers older CMake commands and idioms to avoid and why.
+This book seems to answer the "why" and big picture questions for CMake in a way that no other material has matched.
+It also provides really critical low-level details and tips that are absent in even the official CMake reference documentation.
+This book is self published by the author in a downloadable PDF format with new editions coming out after each new CMake release (i.e. three new editions each year).
+The current edition of the book costs $30 which includes 9 updates for new editions of the book as they come out.
+(Therefore, $30 would cover every new edition over a 3 year period which includes 9 new CMake releases or 6 years if only downloading every other updated edition across 18 CMake releases.)
+The author is not currently a Kitware employee but is a significant contributor to CMake and collaborates closely with Kitware staff.
+The author starts evaluating and updating the book at the beginning of each new CMake release cycle (i.e. right after the first release candidate `vX.Y.0-rc1` is tagged).
```

Comment:
Not sure this and following sentences are needed. Makes for wordiness but not much in the way of truly needed information.

Reply:
Fair enough.  The CC article already says that a new edition is put out after each CMake release.  The process by which new editions of the book are created in relation to CMake releases is not really something our readers need to know.  (The reason I mentioned this was to highlight how embedded Craig Scott is with CMake development and releases.)

Reply:
Addressed in commit https://github.com/betterscientificsoftware/bssw.io/pull/1276/commits/af525be727200d27e6db242b72b4c1fb1ffe61b3